src/electron/views/device-connect-view/device-connect-view.scss

```
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 @import './components/telemetry-permission-dialog.scss';
 @import '../../../reports/components/outcome.scss';
+@import '../../../common/components/cards/fix-instruction-color-box.scss';
```

Can we remove the `../../..` ?
```suggestion
@import 'common/components/cards/fix-instruction-color-box.scss';
```

---

same as before... works for typescript. not sure about scss